fvp.mk

```diff
@@ -69,12 +99,25 @@ TF_A_FLAGS ?= \
 	BL32_EXTRA1=$(OPTEE_OS_PAGER_V2_BIN) \
 	BL32_EXTRA2=$(OPTEE_OS_PAGEABLE_V2_BIN) \
 	BL33=$(EDK2_BIN) \
-	DEBUG=$(DEBUG) \
 	ARM_TSP_RAM_LOCATION=tdram \
 	FVP_USE_GIC_DRIVER=FVP_GICV3 \
 	PLAT=fvp \
 	SPD=opteed
 
+ifeq ($(USE_MEASURED_BOOT),0)
```

Comment:
And here write `ifneq ($(USE_MEASURED_BOOT),y)` rather than `ifeq n`.

Reply:
done